src-tauri/tauri.conf.json

```diff
@@ -0,0 +1,66 @@
+{
+	"$schema": "..\\node_modules/@tauri-apps/cli\\schema.json",
+	"build": {
+		"beforeBuildCommand": "npm run build",
+		"beforeDevCommand": "npm run dev",
+		"devPath": "http://localhost:3000",
+		"distDir": "../frontend/dist"
+	},
+	"package": {
+		"productName": "cornercutter",
+		"version": "0.1.0"
+	},
+	"tauri": {
+		"allowlist": {
+			"all": true
+		},
+		"bundle": {
+			"active": true,
+			"category": "DeveloperTool",
+			"copyright": "",
+			"deb": {
+				"depends": []
+			},
+			"externalBin": [],
+			"icon": [
+				"icons/32x32.png",
+				"icons/128x128.png",
+				"icons/128x128@2x.png",
+				"icons/icon.icns",
+				"icons/icon.ico"
+			],
+			"identifier": "cornercutter.nz",
```

Comment:
is this autogenerated off the name?

Reply:
I had to put something there to build it, so if you have a preferred identifier please let me know :)

Reply:
Just because ideally it wouldn't be nz linked - maybe just remove suffix?